<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-KTDCJLP" name="Scripts.Validlogin$testValidlogin" tests="2" failures="0" timestamp="31 Jul 2019 12:00:00 GMT" time="99.150" errors="2">
  <testcase name="testValidLogin" time="38.238" classname="Scripts.Validlogin$testValidlogin">
    <error type="java.lang.Error" message="Unresolved compilation problems: 
	Workbook cannot be resolved to a type
	WorkbookFactory cannot be resolved
	Cell cannot be resolved to a type
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
	Workbook cannot be resolved to a type
	WorkbookFactory cannot be resolved
	Cell cannot be resolved to a type

	at generics.Excel1.getcellvalue(Excel1.java:17)
	at Scripts.Validlogin$testValidlogin.testValidLogin(Validlogin.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testValidLogin -->
  <testcase name="testValidLogin" time="60.912" classname="Scripts.Validlogin$testValidlogin">
    <error type="java.lang.Error" message="Unresolved compilation problems: 
	Workbook cannot be resolved to a type
	WorkbookFactory cannot be resolved
	Cell cannot be resolved to a type
">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
	Workbook cannot be resolved to a type
	WorkbookFactory cannot be resolved
	Cell cannot be resolved to a type

	at generics.Excel1.getcellvalue(Excel1.java:17)
	at Scripts.Validlogin$testValidlogin.testValidLogin(Validlogin.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testValidLogin -->
</testsuite> <!-- Scripts.Validlogin$testValidlogin -->
